src/sagemaker_server.h

```diff
@@ -159,6 +160,11 @@ class SagemakerAPIServer : public HTTPAPIServer {
 
   static const std::string binary_mime_type_;
 
+  // Type of inference:infer, generate or generate_stream.
```

Comment:
nit: minor re-word - may need line wrapping
```suggestion
  // Triton HTTP handler to map Sagemaker /invocations route to: "infer", "generate", or "generate_stream"
```

Reply:
updated. thanks